src/torrent/data/file_list.cc

```diff
@@ -155,9 +155,6 @@ uint64_t
 FileList::left_bytes() const {
   uint64_t left = size_bytes() - completed_bytes();
 
-  if (left > ((uint64_t)1 << 60))
-    throw internal_error("FileList::bytes_left() is too large.", data()->hash());
-
```

Comment:
You are treating the symptom, not the disease. The above check is there to catch issues in other parts of the code, notably that we don't have a proper check in 'DownloadMain::set_metadata_size' that checks if the size is zero or more than a reasonable size.

In the case of metadata sizes, (1 << 26) or 64Mb seems a good upper limit.